inc/Chair.cpp

```diff
@@ -0,0 +1,69 @@
+#include "Chair.h"
+#include "rapidjson/document.h"
+#include "rapidjson/writer.h"
+#include "rapidjson/stringbuffer.h"
+
+Chair::Chair() : Module(Client()) {}
+Chair::Chair(Client client) : Module(client) {}
+
+void Chair::setInputsJSON(const std::string json)
+{
+  // 1. Parse a JSON string into DOM.
+  rapidjson::Document document;
+  document.Parse(json.c_str());
+
+  this->Motor = document["vM"].GetBool();
```

Comment:
lowerCamelCasing variable names shouldn't start with a capitalized letter. (classes do)

Reply:
setInputsJSON should only write inputs to the cache. Not outputs